spec/rspec/core/formatters/exception_presenter_spec.rb

```diff
@@ -94,7 +94,7 @@ module RSpec::Core
         EOS
       end
 
-      if String.method_defined?(:encoding)
+      if String.method_defined?(:encoding) && !RSpec::Support::OS.windows?
```

Comment:
It was failing?

Reply:
Sorry

Reply:
Not your fault, Windows seems to be flakey on this